docs/handout_en.md

```diff
@@ -138,7 +140,7 @@ virtuální počítač a Linux by měl automaticky naběhnout.
 Ještě jednou vám gratuluji, právě jste prošli tím nejtěžším, co
 začátečníky s Linuxem čeká a občas také odradí. Teď už půjde všechno
 relativně hladce. Jak jsem psal výše, po úspěšném startu systému
-uslyšíte hlášku "Čtečka obrazovky zapnuta". 
```

Comment:
what for?

Reply:
Generally speaking, it is a bad practice to leave random white spaces in your files if it serves no particular reason. 

However, this was done by `vim`, not really intentionally. As it was more hassle to add the white space back in (since I noticed it too late), I kept it removed.

Reply:
As a side note, since the changes are identical, the fact that they are duplicated should not matter in the two PRs.